environments/contrail/roles_data_contrail.yaml

```diff
@@ -276,6 +276,7 @@
     - OS::TripleO::Services::Timezone
     - OS::TripleO::Services::TripleoPackages
     - OS::TripleO::Services::TripleoFirewall
+    - OS::TripleO::Services::NeutronSriovAgent
```

Comment:
do we actually need it? 
As I remember we provide own hiera variables in contrail-vrouter.yaml (dpdk and tsn as well) and have 
include ::tripleo::host::sriov in contrail-tripleo0puppet.